controllers/details.js

```diff
@@ -4,7 +4,8 @@ var api = require('../api');
 
 module.exports = function renderDetails(req, res) {
   if (req.params && req.params.sysnum) {
-    api.requestID(req.params.sysnum)
+    const username = req.headers['X-Auth-Username'] || req.headers['x-auth-username'];
```

Comment:
Headers according to relevant RFC (forgot which one) are supposed to be case insensitive. However delightfully not everyone follows this convention, but some people do....

So http-proxy being used for tests enforces case insensitivity by lowercasing any headers you add automatically, so we need to match on that. But keycloak proxy provides the mixed case header.

I've not been able to come up with a neat way to do case insensitive matching (without iterating over the headers, which is pretty yuck....)